lib/ukdebug/arch/arm64/target.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
```

Comment:
@marcrittinghaus Do you recall where this is from?

Reply:
Looks like both the XML files are from QEMU. Updating the commit

Reply:
I removed the original files from Marc's work because we couldn't find the origin of the `target.xml` files that Marc used. I replaced them with the right target descriptions from the QEMU source tree. The commit that  introduces the `target.xml` files contains both the traceability information that you requested and notes on any changes that I had to make.